src/cpp/dds/domain/DomainParticipantImpl.cpp

```diff
@@ -135,38 +135,58 @@ DomainParticipantImpl::~DomainParticipantImpl()
 }
 
 
-bool DomainParticipantImpl::delete_publisher(
+ReturnCode_t DomainParticipantImpl::delete_publisher(
         Publisher* pub)
 {
+    if(participant_ != pub->get_participant())
```

Comment:
Add a space between `if` and the parenthesis.

Reply:
Changed in PR #604